### ISSUE TITLE
How to get a low latency public playback URL?

### ISSUE THREAD

**Issue by angelfor3v3r** [datarhei/restreamer#619]
**Subject of the issue**
I'm able to stream with RTMP/SRT and play it back fine in VLC but I was wondering how I can get a public stream URL for SRT/RTMP to a stream that won't allow people to stream to it.

The password seems like a global value and doesn't seem useful since people who want to play the SRT stream can just change "mode:request" to "mode:publish", and stream anyway on someone elses "channel" when they're streaming/not streaming.

I'm very new to this, but I would love to know how to achieve this. Perhaps the only way is "restreaming" to some 3rd party app somehow? I'm really not sure. Thanks!

**Your environment**
I'm using the latest version (2.6.0) with docker on an x86-64 CPU.

**Files**
I don't think this is necessary.

**Steps to reproduce**
* Start a stream.
* How to get a public link?

**Expected behavior**
I can find a public RTMP/SRT playback URL.

**Actual behavior**
I can't find one.

**Comment by ioppermann** [comment]
This is true, there's currently only one token configurable on a RTMP/SRT stream, which makes it difficult to share. We're working on a solution for that.
